packages/@okta/vuepress-site/docs/reference/api/captchas/index.md

```diff
@@ -0,0 +1,922 @@
+  "secretKey": "copy_your_secret_key",
+  "type": "RECAPTCHA_V2"
+}' "https://${yourOktaDomain}/api/v1/captchas"
+```
+
+##### Response
+
+```http
+HTTP/1.1 201 Created
+Content-Type: application/json
+```
+
+```json
+{
+  "id": "cap18c2Ey3iR9BLDb0g4",
+  "name": "myReCaptcha",
+  "siteKey": "6LeVzVIaAAAAAHCbhZ-uxTihGl7iuufxEF_dMH-x",
+  "type": "RECAPTCHA_V2",
+  "_links": {
+    "self": {
+      "href": "http://${yourOktaDomain}/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+
+#### Error example
+The following request would return an error when the org already has a CAPTCHA.
+
+##### Request
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "myReCaptcha2",
+  "siteKey": "copy_your_site_key",
+  "secretKey": "copy_your_secret_key",
+  "type": "RECAPTCHA_V2"
+}' "https://${yourOktaDomain}/api/v1/captchas"
+```
+
+##### Response
+```http
+HTTP/1.1 403 Forbidden
+Content-Type: application/json
+```
+
+```json
+{
+  "errorCode": "E0000166",
+  "errorSummary": "CAPTCHA count limit reached. At most one CAPTCHA instance is allowed per Org. Please remove existing CAPTCHA to create a new one.",
+  "errorLink": "E0000166",
+  "errorId": "oaeujx4SxU6S-Gc6HvPnpG2MQ",
+  "errorCauses": [
+    {
+      "errorSummary": "CAPTCHA count limit reached. At most one CAPTCHA instance is allowed per Org. Please remove existing CAPTCHA to create a new one.",
+      "reason": "EXCEEDS_MAX_VALUE",
+      "locationType": "url",
+      "domain": "captcha"
+    }
+  ]
+}
+```
+
+### Get CAPTCHA
+
+<ApiOperation method="get" url="/api/v1/captchas/{id}" />
+
+Fetches a CAPTCHA by `id`.
+
+#### Request path parameters
+
+| Parameter | Type        | Description   |
+| --------- | ----------- | ------------- |
+| `id `       | String (URL)        | Required. ID of a CAPTCHA. |
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+N/A
+
+#### Response body
+
+The requested [CAPTCHA Response](#captcha-response-object)
+
+#### Use example
+
+This example returns a CAPTCHA object by id:
+
+##### Request
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/captchas/{id}"
+```
+
+##### Response
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+```
+
+```json
+{
+  "id": "cap18c2Ey3iR9BLDb0g4",
+  "name": "myReCaptcha",
+  "siteKey": "6LeVzVIaAAAAAHCbhZ-uxTihGl7iuufxEF_dMH-x",
+  "type": "RECAPTCHA_V2",
+  "_links": {
+    "self": {
+      "href": "http://${yourOktaDomain}/api/v1/captchas/cap18c2Ey3iR9BLDb0g4",
+      "hints": {
+        "allow": [
+          "GET",
+          "POST",
+          "PUT",
+          "DELETE"
+        ]
+      }
+    }
+  }
+}
+```
+
+### Get all CAPTCHAs
```

Comment:
Do we even need to document this one? Seems a little counterintuitive since we only have support for one CAPTCHA, but I understand including it for completeness.

Reply:
This one is not just for the multiple captcha instances in the future, it also have a benefit:
User don't need to provide captcha id to retrieve the captcha information, it's very convenient to use.

For example, if user needs to update, get or delete a captcha from API. Without this get all method, they need to go back to UI to copy captcha Id then come back to  use API.

With this get all method, it is very convenient for user to get captcha id and this enables use to write their own programs to manage captcha by consuming apis.

Reply:
Makes sense, thanks for the additional context, @henryli-okta !